src/test/java/BaseTest.java

```diff
@@ -1,85 +1,77 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
 import org.openqa.selenium.safari.SafariDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
 import org.testng.annotations.AfterMethod;
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Parameters;
 
+import java.net.MalformedURLException;
+import java.net.URI;
 import java.time.Duration;
 import java.util.UUID;
 
 
 public class BaseTest {
+
     public static WebDriver driver = null;
     public static String url = "https://bbb.testpro.io/";
+    public static WebDriverWait wait;
+    public static Actions actions;
 
-
-    @BeforeSuite
-    static void setupClass() {
-        WebDriverManager.chromedriver().setup();
-    }
+//    @BeforeSuite
+//    public static void setupClass() {
+//        WebDriverManager.chromedriver().setup();
+//    }
 
     @BeforeMethod
-    public static void launchBrowser() {
-        LoginTests.driver = new ChromeDriver();
-        LoginTests.driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+    @Parameters ({"BaseURL"})
+    public static void openBrowser(String BaseURL) throws MalformedURLException {
+        driver = pickBrowser(System.getProperty("browser"));
```

Comment:
I would do it in another way, but your solution is also fine 

For example

`driver = pickBrowser("browser");`